IFComp/lib/IFComp/Controller/Play.pm

```diff
@@ -36,6 +36,7 @@ sub fetch_entry :Chained('root') :PathPart('') :CaptureArgs(1) {
             || ( $c->stash->{ current_comp }->status eq 'processing_votes' )
             || ( $c->stash->{ current_comp }->status eq 'over' )
             || ( $c->user && ( $entry->author->id eq $c->user->id ) )
+            || ( $c->check_user_roles( 'curator' ) )
```

Comment:
The check_user_roles() method will use $c->user by default, and it will return false if there is no user.

See both assert_user_roles and check_user_roles here: https://metacpan.org/pod/Catalyst::Plugin::Authorization::Roles#assert_user_roles-user-roles